presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcConnection.java

```diff
@@ -235,6 +235,17 @@ public void testApplicationName()
         }
     }
 
+    @Test
+    public void testClientTags()
```

Comment:
Move this test to be directly below `testExtraCredentials` so that the ordering is consistent everywhere.